specific/cmo_sale_discount_total_enhance/models/sale.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+
+from openerp import api, fields, models
+import openerp.addons.decimal_precision as dp
+
+
+class sale_order(models.Model):
+    _inherit = 'sale.order'
+
+    discount_type = fields.Selection([
```

Comment:
If you only change some property, only set that property
(no need to bring on all the properties)